models/zendesk__ticket_field_history.sql

```diff
@@ -82,7 +132,7 @@ with change_data as (
         , case when  cast( {{ col.name }} as {{ dbt_utils.type_string() }} ) = 'is_null' then null else {{ col.name }} end as {{ col.name }}
         {% endfor %}
 
-    from joined
+    from fill_values
```

Comment:
Final Output:

The below is the **NEW** logic if the last incremental run was on 2021-09-28. You can see the model will pass through the values accurately and upsert into the `zendesk__ticket_field_history` model
![image](https://user-images.githubusercontent.com/74217849/135201232-17b9161b-1cb3-4912-8e73-22d19bcc067c.png)

Reply:
The below is the **OLD** logic if the last incremental run was on 2021-09-28. You can see the incremental logic does not work
![image](https://user-images.githubusercontent.com/74217849/135200481-d45d84b8-df61-4b51-bbeb-c3e921783332.png)